store/mockstore/mocktikv/mppserver.go

```diff
@@ -0,0 +1,54 @@
+// Copyright 2019 PingCAP, Inc.
```

Comment:
It's better to rename the file name to `coprocessor.go` or another name.

Reply:
ok